### DISCUSSION TITLE
Stereo Tool - Not working?

### DISCUSSION THREAD

**Discussion by thehijacker** [AzuraCast/AzuraCast#5530] [Q&A]
Hello,

Stereo Tool command line binary uploaded and it says it is using latest version (9.91). Under my station profile under Auto DJ tab I have set Audio Processing Method to Stereo Tool. After restarting station I see no change in output quality. Also nowhere in the logs I can see Stereo Tool beeing used.

Am I a missing some step somewhere?

Thank you and best regards.

**Comment by Vaalyn** [comment]
You'll need to upload a Stereo Tool Configuration file which defines what kind of audio processing Stereo Tool should apply.
You can find this on the Station in the Broadcasting -> Upload Stereo Tool Configuration page.

**Reply by thehijacker** [reply]
That makes sense. Is there a sample config somewhere?
